_pages/en_US/restoring-nand.md

```diff
@@ -49,7 +49,17 @@ It is very important to test that your NAND backup is working before attempting
 8. Click `OK`
 9. Launch any Nintendo DS ROM (`.nds` file)
 
-If no$gba loads the DSi menu, then continue to the next section. If it shows any kind of error ***do not flash that backup***!
+If no$gba loads the DSi menu (or the Unlaunch Filemenu), then continue to the next section. If it shows any kind of error ***do not flash that backup***!
+
+## Uninstalling Unlaunch from your NAND backup (optional)
+Follow this if you dumped your NAND backup after you installed Unlaunch and you would like to uninstall Unlaunch from your system.
+1. Download the latest version of the [Unlaunch installer](https://problemkaputt.de/unlaunch.zip)
+1. Extract `UNLAUNCH.DSI` from `unlaunch.zip`
+1. Launch `UNLAUNCH.DSI` in no$gba and start it from the Game Card slot
```

Comment:
If no$gba has been closed and re-opened since the last section then it'll directly load the Unlaunch installer instead of the DSi menu, which is fine but should maybe be accounted for here.